src/ci/resources/tes_application.conf

```diff
@@ -52,8 +57,14 @@ backend {
             container: "cromwell"
             endpoint: "https://<storage-account>.blob.core.windows.net"
             subscription: "00001111-2222-3333-aaaa-bbbbccccdddd"
-            workspace-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
-            b2cToken: "Zardoz"
+            # WSM config is needed for accessing WSM-managed blob containers
+            # created in Terra workspaces.
+            workspace-manager {
+              url: "https://workspace.dsde-dev.broadinstitute.org"
+              workspace-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+              container-resource-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+              b2cToken: "Zardoz"
+            }
```

Comment:
With the new config structure I don't think this needs to be in here twice anymore.